src/main/java/com/sprints/SprintsApiApplication.java

```diff
@@ -3,12 +3,27 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 
+import com.sprints.interceptors.UndeclaredParamsHandlerInterceptor;
+
+@SuppressWarnings("deprecation")
 @SpringBootApplication(exclude = {DataSourceAutoConfiguration.class })
 public class SprintsApiApplication {
 	public static void main(String[] args) {
 		SpringApplication.run(SprintsApiApplication.class, args);
 		    
-		
+	}
+	
+	@Configuration
+	public static class WebMvcConfig extends WebMvcConfigurerAdapter  {
```

Comment:
Why we need this method ?

Reply:
That class is needed as configuration so I can register the Interceptors for the undeclared params in the Get filters